### ISSUE TITLE
[Feature Request]: re-add .devcontainer definition

### ISSUE THREAD

**Issue by JonasPammer** [JonasPammer/ansible-role-bootstrap#32]
### Proposed feature

with dind/dfd

### Rationale

see DEVELOPMENT.adoc

**Comment by JonasPammer** [comment]
it's almost working with the bare-minimum Docker in Docker Dev Container:
```
$ sudo apt update -y && sudo apt upgrade -y
$ sudo apt install python3 python3-pip python3-setuptools
$ sudo python3 -m pip install --upgrade pip && sudo python3 -m pip install tox
$ sudo tox  # yes, i tried without sudo.
....
  py3-ansible-2.9: commands succeeded
  py3-ansible-2.10: commands succeeded
  py3-ansible-2.11: commands succeeded
ERROR:   py3-ansible-2.12: could not install deps [ansible-core == 2.12.*, molecule[docker], docker == 4.*]; v = InvocationError("/workspaces/ansible-role-bootstrap/.tox/py3-ansible-2.12/bin/python -m pip install 'ansible-core == 2.12.*' 'molecule[docker]' 'docker == 4.*'", 1)

$ pip index versions ansible-core
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
ansible-core (2.11.9)
Available versions: 2.11.9, 2.11.8, 2.11.7, 2.11.6, 2.11.5, 2.11.4, 2.11.3, 2.11.2, 2.11.1, 2.11.0

$ neofetch
       _,met$$$$$gg.          vscode@8e4ed10f78fa 
    ,g$$$$$$$$$$$$$$$P.       ------------------- 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 10 (buster) on Windows 10 x86_64 
 ,$$P'              `$$$.     Kernel: 5.10.16.3-microsoft-standard-WSL2 
',$$P       ,ggs.     `$$b:   Uptime: 36 mins 
`d$$'     ,$P"'   .    $$$    Packages: 344 (dpkg) 
 $$P      d$'     ,    $$P    Shell: bash 5.0.3 
 $$:      $$.   -    ,d$$'    Terminal: vscode 
 $$;      Y$b._   _,d$P'      CPU: 11th Gen Intel i9-11900H (16) @ 2.496GHz 
 Y$$.    `.`"Y$$$$P"'         Memory: 1472MiB / 15880MiB 
 `$$b      "-.__
  `Y$$                                                
   `Y$$.
     `$$b.
       `Y$$b.
          `"Y$b._
              `"""
```

**Comment by JonasPammer** [comment]
research concluded the minimum python for ansible 2.12 is python 3.8
